MeshMan/Hangman/Hangman.swift

```diff
@@ -0,0 +1,194 @@
+//
+//  Hangman.swift
+//  MeshMan
+//
+//  Created by Russell Pecka on 12/28/17.
+//  Copyright © 2017 Russell Pecka. All rights reserved.
+//
+
+import Foundation
+import MultipeerConnectivity
+
+// MARK: - HangmanDelegate
+
+protocol HangmanDelegate: class {
+    func hangman(_ hangman: Hangman, stateUpdatedFromOldState oldState: HangmanGameState, toNewState newState: HangmanGameState, obfuscationResult: Hangman.WordObfuscationPayload)
+    func hangman(_ hangman: Hangman, endedGameWithConclusion conclusion: Hangman.Conclusion)
```

Comment:
This delegate call is unneeded, just decide the conclusion when updating the state